mergify_engine/actions/merge/helpers.py

```diff
@@ -68,11 +69,19 @@ def merge_report(pull, strict):
     return conclusion, title, summary
 
 
-WAIT_FOR_CI_REPORT = (
-    None,
-    "Base branch updates done",
-    "The pull request has been automatically updated to follow its base branch and will be merged soon",
-)
+def get_wait_for_ci_report(pull):
+    queue_status = ""
+    pulls = queue.get_pulls_from_queue(pull)
+    if pulls:
+        links = ", ".join((f"#{pull}" for pull in pulls))
+        queue_status = f"The following pull requests are queued: {links}"
+
+    return (
+        None,
+        "Base branch updates done",
+        "The pull request has been automatically updated to follow its base branch and "
+        f"will be merged soon.\n\n{queue_status}",
```

Comment:
Do we need the `\n\n` if the `queue_status` is empty?

Reply:
It's to have 2 paragraphes

Reply:
Right my question was, do we need 2 paragraphs when the second one is empty? ;)